### ISSUE TITLE
please add for whitelist engram

### ISSUE THREAD

**Issue by ADXX230** [VoidArkMods/QoLPlus#92]
hey can you please add the following below to this list https://github.com/VoidArkMods/QoLPlus/blob/main/EngramEntries so we can white list engrams we want to use

need to add 
QOL+ converter
QOL+ PRESERVING BIN
QOL+ BLUEPRINT MAKER
QOL+ FABRICATOR
QOL+ REFRIGERATOR
QOL+ INDUSTRIAL GRINDER
QOL+ INDUSTRIAL FORGE
QOL+ TEK REPLICATOR

and any new stuff comes out in the future, thank you

**Comment by bboerni2** [comment]
white listing working already, like it did with s+

**Comment by ADXX230** [comment]
yeah its working for me as well but i dont know the engram entries for list as above to put into the list, the only ones i know what to put into the command are what in this link https://github.com/VoidArkMods/QoLPlus/blob/main/EngramEntries
